docs/docs/03-language-reference.md

```diff
@@ -1186,6 +1184,19 @@ The loop invariant in for loops is implicitly re-assignable (`var`).
 
 ---
 
+### 2.8 throw
+
+The **throw** statement throws a user-defined exception.
+Execution of the current function will stop (the statements after throw won't be executed), and control will be passed to the first catch block in the call stack.
+If no catch block exists among caller functions, the program will terminate.
```

Comment:
Should we specify what "terminate" means depending on phase? e.g. throwing in preflight causes a compilation failure.

Reply:
Added